### ISSUE TITLE
<script> tag inside window.___DATA___ makes this object broken

### ISSUE THREAD

**Issue by Grawl** [universal-vue/uvue#46]
**Describe the bug**

With `</script>` in `window.__DATA__` there is a parse error:

```
Uncaught SyntaxError: Invalid or unexpected token
```

**To Reproduce**

Make your state look like this:

```html
<script>
window.__DATA__ = {
	'components': [
		{
			'body': 'some HTML with <script src="http://some-url"></script>',
		},
	],
	'state': {
		'rows': [],
	},
}
</script>
```

And you will see this HTML is not valid because `<script>` is closed in the middle of it's content'

Seems like the problem is here:

https://github.com/universal-vue/uvue/blob/c6c421b65151481aafa6523d72b8293b331cfb22/packages/%40uvue/server/src/Renderer.ts#L43

**Expected behavior**

State loaded without runtime errors

**Additional context**

```
node: 12.15.0
vue-cli: 3.12.1
@uvue/core: 0.1.5
@uvue/server: 0.1.3
os: macOS Catalina 10.15.2 (19C57)
browser: Google Chrome 80.0.3987.122
```

**Comment by Grawl** [comment]
It's not clear that HTML is broken because of github code highlighter.

If you copy this code into WebStorm, you will see the problem:

<img width="905" alt="изображение" src="https://user-images.githubusercontent.com/846774/75771464-c2ec5580-5d95-11ea-8bc7-7953f136ab55.png">

**Comment by Grawl** [comment]
As a workaround, I will replace all `<script>` tags to `[script]` and replace back on client.

**Comment by chymz** [comment]
Fixed in `0.1.6`
